<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationExport</class>
 <widget class="QWidget" name="ConfigurationExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>HTML Behavior</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="RoundCorners">
        <property name="text">
         <string>Use &amp;rounded corners on date/timestamp header</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SystemColors">
        <property name="text">
         <string>Use system &amp;colors in exported HTML files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="IncludeTags">
        <property name="text">
         <string>Include &amp;tags in exported entries</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Preferred HTML Fonts</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select header font:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFontComboBox" name="HeaderFont">
        <property name="toolTip">
         <string>&lt;p&gt;Recommendation: For best results, use sans serif fonts (e.g. Arial, Verdana, Calibri) for headers. The preferred font is used by default; other fonts are substituted if the preferred font is not available on the current system.&lt;/p&gt;</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>6</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Select body text font:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFontComboBox" name="BodyFont">
        <property name="toolTip">
         <string>&lt;p&gt;Recommendation: Serif fonts (e.g. Times New Roman, Cambria, etc.) work best with body text. The preferred font is used by default; other fonts are substituted if the preferred font is not available on the current system.&lt;/p&gt;</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Font Sizes and Units</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Header font size/unit</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="HeaderSize">
          <property name="minimumSize">
           <size>
            <width>52</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>18</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>22</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>24</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>26</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>28</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>30</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>34</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>40</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="HeaderUnit">
          <property name="toolTip">
           <string>Em units scale relative to font size; Pt units are fixed point size.</string>
          </property>
          <item>
           <property name="text">
            <string>pt</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>em</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Body font size/unit</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="BodySize">
          <property name="minimumSize">
           <size>
            <width>52</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="BodyUnit">
          <property name="toolTip">
           <string>Em units scale relative to font size; Pt units are fixed point size.</string>
          </property>
          <item>
           <property name="text">
            <string>pt</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>em</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="PreviewFonts">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Click to see a sample document rendered with the current settings. All font/size changes are temporary until you click the &lt;span style=&quot; font-weight:600;&quot;&gt;OK&lt;/span&gt; button below.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Preview Export Settings</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/icons/document-globe.png</normaloff>:/icons/document-globe.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;FYI: Changes will &lt;span style=&quot; font-weight:600;&quot;&gt;not&lt;/span&gt; affect existing documents.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
